stuff/pipelines/rabbitmq_database.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+
+'''Mysql consults to generate messages'''
+
+import datetime
+from typing import Any, Dict, List, Optional
+import mysql.connector
```

Comment:
```suggestion

import mysql.connector
```

hay que poner una línea vacía entre imports built-in y de pypi.

Reply:
Listo